docs/docsite/rst/playbooks_loops.rst

```diff
@@ -277,10 +277,12 @@ The authorized_key pattern is exactly where it comes up most.
 Looping over Integer Sequences
 ``````````````````````````````
 
-``with_sequence`` generates a sequence of items in ascending numerical order. You
-can specify a start, end, and an optional step value.
+``with_sequence`` generates a sequence of items. You
+can specify a start value, an end value, an optional "stride" value that specifies the number of steps to increment the sequence, and an optional printf-style format string.
 
-Arguments should be specified in key=value pairs.  If supplied, the 'format' is a printf style string.
+Arguments should be specified as key=value pair strings.
```

Comment:
looked at code, you are correct, this was victim of 'blanket' yamliziation of examlpes